experimental/dds/tree2/src/feature-libraries/default-schema/defaultChangeFamily.ts

```diff
@@ -85,7 +87,10 @@ export interface IDefaultEditBuilder {
 	 * The returned object can be used (i.e., have its methods called) multiple times but its lifetime
 	 * is bounded by the lifetime of this edit builder.
 	 */
-	sequenceField(field: FieldUpPath): SequenceFieldEditBuilder;
+	sequenceField(
+		field: FieldUpPath,
+		shapeInfo?: { schema: StoredSchemaCollection; policy: FullSchemaPolicy },
```

Comment:
this parameter needs documentation. I think this is only intended as a temporary thing until the encoded format is properly separated? Be sure to clarify that.

Reply:
Updated sequenceField doc comment